src/Tree.js

```diff
@@ -64,5 +66,10 @@ export default class Tree extends React.Component {
 Tree.propTypes = {
   nodes: PropTypes.arrayOf(PropTypes.shape(FlattenedNode)).isRequired,
   children: PropTypes.func.isRequired,
-  onChange: PropTypes.func.isRequired
+  onChange: PropTypes.func.isRequired,
+  leftMarginAmount: PropTypes.number,
+};
+
+Tree.defaultProps = {
+  leftMarginAmount: 30
```

Comment:
Just not sure about the name, shall we call it something that is more straight forward to understand? Something like nodeMarginLeft?

Reply:
The default prop shouldn't be in here as this class is not exported, it should be in `TreeContainer`